src/RemoteHandler/DataReceiver.js

```diff
@@ -57,8 +70,25 @@ class DataReceiver extends RemoteHandler {
         }, errorCallback);
     }
 
-    fetchEventTypesOf(productId, successCallback, errorCallback) {
-        const APIRoute = DataReceiver.getAPIRouteForEventTypesOfProduct().format(productId);
+    fetchConfiguration(configurationId, successCallback, errorCallback) {
+        const APIRoute = DataReceiver.getAPIRouteForProductConfiguration().format(configurationId);
+        const URI = DataReceiver.getServerRequestURI().format(this.remoteAddress, this.remotePort, APIRoute);
+        const callbackContainer = {
+            "dataMappingFunction":    this.dataMapper.mapConfigurations,
+            "clientSuccessCallback":  successCallback,
+            "clientErrorCallback":    errorCallback
+        };
+        this.client.addRequest(URI, this.requestMethod, callbackContainer);
+    }
+
+    fetchEventTypesOf(productId, successCallback, errorCallback, forProduct=true) {
+        var APIRoute = "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "APIRoute" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1854)